src/utils/postParser.js

```diff
@@ -82,6 +82,15 @@ export const parseComments = async (comments) => {
     comment.markdownBody = get(comment, 'body');
     comment.body = renderPostBody(comment, true, webp);
 
+    //parse json meta;
+    if (typeof comment.json_metadata === 'string' || comment.json_metadata instanceof String) {
+      try {
+        comment.json_metadata = JSON.parse(comment.json_metadata);
+        comment.json_metadata.tags = ['hive-123123'];
```

Comment:
Please check this. IF tags is empty we set default tag, not always. And it has to be `['hive-125125']` (Ecency community) not 123123...